Mathlib/MeasureTheory/Measure/Haar/Unique.lean

```diff
@@ -272,8 +272,7 @@ lemma exists_integral_isMulLeftInvariant_eq_smul_of_hasCompactSupport (μ' μ :
     integral_isMulLeftInvariant_isMulRightInvariant_combo f_cont f_comp g_cont g_comp g_nonneg g_one
   /- Since the `ν`-factor is the same for `μ` and `μ'`, this gives the result. -/
   rw [← A, mul_assoc, mul_comm] at B
-  simp only [B, integral_smul_nnreal_measure]
-  rfl
+  simp [B, integral_smul_nnreal_measure, c, NNReal.smul_def]
```

Comment:
I'm slightly paranoid now: does this make a performance difference? (Can you e.g. check locally? If the simp is not very slow, e.g. does not show up with `set_option trace.profiler before the declaration, and reloading the file, this should be fine.)